### PR TITLE
added info and warn box with respect to resolver configuration

### PR DESCRIPTION
**Pull request by Eknir**

### CODE REVIEW

File: docs/getting-started/host-your-website-using-ENS.md [hunk 1]
```diff
@@ -17,6 +17,12 @@ In order to resolve ENS names using your API endpoints, you must specify a valid
 bee start --resolver-options "https://cloudflare-eth.com"
 ```
 
+:::info
+Use the `resolver-options` flag to set point the bee resolver to any ens-compatible smart-contract on any evm-compatible chain
```
Comment (attila-lendvai) [inline docs/getting-started/host-your-website-using-ENS.md:21]:
```suggestion
Use the `resolver-options` flag to point the bee resolver to any ENS compatible smart-contract on any EVM compatible chain
```

File: docs/getting-started/host-your-website-using-ENS.md [hunk 2]
```diff
@@ -17,6 +17,12 @@ In order to resolve ENS names using your API endpoints, you must specify a valid
...
+:::
+:::warn
+When your bee node, or the bee node of a gateway that you are interacting with is not connected to the standard EVM contract on Ethereum mainnet, familiar names (e.g. `swarm.eth`) may resolve to different content.
```
Comment (attila-lendvai) [inline docs/getting-started/host-your-website-using-ENS.md:24]:
```suggestion
When your bee node, or the bee node of a gateway that you are interacting with, is not connected to the standard EVM contract on Ethereum mainnet, then familiar names (e.g. `swarm.eth`) may resolve to different content.
```

**Review by attila-lendvai — CHANGES_REQUESTED** [review]

**Comment by Eknir** [comment]
@attila-lendvai , Thanks for the proposals! Could you please have a look again?

File: docs/getting-started/host-your-website-using-ENS.md [hunk 3]
```diff
@@ -17,6 +17,12 @@ In order to resolve ENS names using your API endpoints, you must specify a valid
 bee start --resolver-options "https://cloudflare-eth.com"
 ```
 
+:::info
+Use the `resolver-options` flag to point the bee resolver to any ENS compatible smart-contract on any EVM compatible chain
+:::
+:::warn
+When your bee node, or the bee node of a gateway that you are interacting with, is not connected to the standard EVM contract on Ethereum mainnet, then familiar names (e.g. `swarm.eth`) may resolve to different content.
```
Comment (significance) [inline docs/getting-started/host-your-website-using-ENS.md:24]:
```suggestion
Make sure you trust the gateway you are interacting with. To be sure you are retrieving the correct content run your own ENS resolver and Bee node. 
```

**Review by significance — APPROVED** [review]

File: docs/getting-started/host-your-website-using-ENS.md [hunk 4]
```diff
@@ -18,10 +18,10 @@ bee start --resolver-options "https://cloudflare-eth.com"
 ```
 
 :::info
-Use the `resolver-options` flag to set point the bee resolver to any ens-compatible smart-contract on any evm-compatible chain
+Use the `resolver-options` flag to point the bee resolver to any ENS compatible smart-contract on any EVM compatible chain
 :::
 :::warn
-When your bee node, or the bee node of a gateway that you are interacting with is not connected to the standard EVM contract on Ethereum mainnet, familiar names (e.g. `swarm.eth`) may resolve to different content.
+Make sure you trust the gateway you are interacting with. To be sure you are retrieving the correct content run your own ENS resolver and Bee node.
```
Comment (attila-lendvai) [inline docs/getting-started/host-your-website-using-ENS.md:24]:
```suggestion
Make sure you trust the gateway you are interacting with! To ensure that you are retrieving the correct content, run your own ENS resolver and Bee node.
```

**Review by attila-lendvai — CHANGES_REQUESTED** [review]

**Comment by attila-lendvai** [comment]
@Eknir AFAIK, you can accept these "request changes" suggestions that i'm creating with a single click, right here on github. let me know if your workflow includes more hassle than that!

**Review by attila-lendvai — APPROVED** [review]

**Pull request merged**